onward/app/controllers/package.scala

```diff
@@ -0,0 +1,13 @@
+import slices.FixedContainers
+
+package object controllers {
```

Comment:
I don't really like Mixins. Overuse of them slows down the compiler a lot, and there's no reason to use them over a simple package with a function in it, unless the Mixin itself is stateful, which none of ours are.